src/sparsezoo/v2/objects/model_directory.py

```diff
@@ -422,25 +464,13 @@ def _download(
         self, file: Union[File, List[File], Dict[Any, File]], directory_path: str
     ) -> bool:
 
-        # TODO: This is a hack for now,
-        #  some files cannot be downloaded if
-        #  ModelDirectory created .from_zoo_api()
-
-        SKIP_ATTRIBUTES = [
-            "framework-files",
-            "analysis.yaml",
-            "benchmarks.yaml",
-            "eval.yaml",
-            "recipe_foo.md",
-            "recipe_bar.md",
-            "sample-labels.tar.gz",
-        ]
         if isinstance(file, File):
-            # TODO: Continuing with the hack
-            if file.name in SKIP_ATTRIBUTES:
-                return True
-
-            if file.url:
+            if file.url or (
+                file.url is None
+                and any(
+                    [isinstance(file, _class) for _class in [Directory, NumpyDirectory]]
```

Comment:
Allow early exit for `any`
```suggestion
                    isinstance(file, _class) for _class in [Directory, NumpyDirectory]
```